hiring/README.md

```diff
@@ -0,0 +1,41 @@
+# Our hiring process
+
+We believe that our hiring process should be [fair and repeatable](rationale.md).
+
+## Pipeline
+
+ - Defining a need and expectation for hiring
+ - Screening
+ - Interview & Pairing
+ - Hire
+ - Nurture
+
+Our pipeline is tracked on a [trello board](https://trello.com/b/0kmnv5Wo/recruitment).
+
+### Defining a need and expectation for hiring
+
+As described in our [Roles at Made Tech](../roles/README.md) document, candidates must be judged against a specific role with the needs and expectations of that role explicitly defined. This allows us to judge candidates like for like and also keeps us focussed on the reason for a hire.
+
+### Screening
+
+Our initial screening stage is usually a phone interview for a brief conversation to see where a candidate is at in their careers, talk about Made and perhaps have a technical chat about past experiences.
+
+Without clear examples of engineering competency a candidate may be asked to complete our [remote test](remote_test/README.md).
+
+### Interview & Pairing
+
+The candidate is invited into our offices for an hour and a half. Usually the first 30 minutes will be spent talking more in depth about the candidates previous experience of processes and practices along with a more detailed explanation of how we work.
+
+After this introduction we usually pair with a candidate on a problem for 30 minutes. This step is used for seeing how well a candidate communicates and works with us. Of course it is a pressured environment, however we try and make the candidate at ease by sharing some of the work and discussing the problem with them.
+
+We then follow up with some more technical, deeper thinking questions documented in this [spreadsheet](https://docs.google.com/spreadsheets/d/1gzOeu_a9uPdOA0roY1oMo_S4pLQCCdrysgyFANcu0FA/edit#gid=1588607612). (Sorry if you're a candidate looking at this, we do not handout these questions in advance.)
+
+Finally we answer any questions the candidate may have.
+
+### Hire
+
+Ideally at this point the candidate is great and we hire them. If this is the case we will prepare a contract and send it to them as soon as possible. After this the [on-boarding process](../company/first_day.md) begins.
+
+### Nurture
+
+In the case we see a candidate as promising but does not fulfil our current need for hiring we will add them to our nurture list where we stay in touch. We will often invite candidates to events or even offer them mentorship. We understand that networking is a powerful tool for our hiring pipeline.
```

Comment:
"In the case where we see a candidate as promising, but where they do not fulfil our current needs for hiring.." or similar?